processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl

```diff
@@ -43,20 +43,16 @@
 -->
 <#macro callTargetWriteAccessor>
     <@lib.handleNullCheck>
-      <#if directAssignment>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( <@wrapLocalVarInCollectionInitializer/> );
-    <#else>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( ${localVarName} );
-    </#if>
```

Comment:
@agudian : this is spotted by Checkstyle on the generated code. However, the current situation (leading up to this Issue / PR) isn't. Can we catch the current situation as well?